### DISCUSSION TITLE
Add group to EA

### DISCUSSION THREAD

**Discussion by nicole-ge** [pnp/powershell#4320] [Q&A] [answered]
Hi community,

Is there a possibility with pnp to add an azure ad group to an enterprise application?

**Comment by nicole-ge** [comment]
Hi @martinlingstuyl 
Maybe do you have any idea

**Reply by martinlingstuyl** [reply]
Hi @nicole-ge, I don't think there is, no. Sounds like a nice idea to build though... 😀

**Reply by nicole-ge** [reply]
Hi @martinlingstuyl , how I can help to build or realize this cmdlet? 
Because this is an important use to add ad user or ad group to an enterprise application with pnp module

**Reply by martinlingstuyl** [reply, marked as answer]
What you could do for starters is create an issue in the [PnP PowerShell issues list](https://github.com/pnp/powershell/issues). Describe what you want as detailed as possible. After doing that maintainers may have some feedback and someone (or you 😁) may take it up to build it.
